### PR TITLE
864 add email reviewer

### LINKED ISSUE #864: [TASK] Add email to admin/reviewers endpoint in Contributions Catalog
### Description
Add email id to reviewers collection. A reviewer record should have name, githubUsername and email id.

### Acceptance Criteria
Email id field in reviewers collection.

### PR DESCRIPTION
**Pull request by minump**
## Description
Adds email field in reviewer collection. Adds email field in admin/reviewers endpoint in Contributions Catalog

**Fixes #864**

[comment]: # (This project only accepts pull requests related to open issues.)
[comment]: # (If suggesting a new feature or change, please discuss it in an issue first.)
[comment]: # (If fixing a bug, there should be an issue describing it with steps to reproduce.)

## Review Time Estimate
_Please give your idea of how soon this pull request needs to be reviewed by selecting one of the options below. This can be based on the criticality of the issue at hand and/or other relevant factors._

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Immediately
- [x] Within a week
- [ ] When possible

## Type of changes
_Please select a relevant option:_

[comment]: # (To select an option, please put an 'x' in the applicable box.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] Bug fix (non-breaking change which fixes an issue).
- [x] New feature (non-breaking change which adds functionality).
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected).
- [ ] Other (any another change that does not fall in one of the above categories.)

## Checklist:
_Please select all applicable options:_

[comment]: # (To select your options, please put an 'x' in the all boxes that apply.)
[comment]: # (If you're unsure about any of these, don't hesitate to ask. We're here to help!.)

- [ ] I have signed the Rokwire Contributor License Agreement ([CLA](https://rokwire.org/rokwire_cla)). (_Any contributor who is not an employee of the University of Illinois whose official duties include contributing to the Rokwire software, or who is not paid by the Rokwire project, needs to sign the CLA before their contribution can be accepted._)
- [x] I have updated the [CHANGELOG](../CHANGELOG.md).
- [ ] I have read the [Contributor Guidelines](../CONTRIBUTING.md).
- [x] I have performed a self-review of my own code.
- [x] I have commented my code, particularly in hard-to-understand areas.
- [ ] My change requires updating the documentation.
- [ ] I have made necessary changes to the documentation.
- [ ] I have added tests related to my changes.
- [ ] My changes generate no new warnings.
- [ ] New and existing unit tests pass locally with my changes.
- [ ] Any dependent changes have been merged and published in downstream modules.

---

[comment]: # (Template credit: This pull request template is based on Embedded Artistry {https://github.com/embeddedartistry/templates/blob/master/.github/PULL_REQUEST_TEMPLATE.md}, Clowder {https://github.com/clowder-framework/clowder/blob/develop/.github/PULL_REQUEST_TEMPLATE.md}, and TalAter {https://github.com/TalAter/open-source-templates} templates.)

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -465,4 +466,4 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
 [1.0.3]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.2...1.0.3
 [1.0.2]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.1...1.0.2
 [1.0.1]: https://github.com/rokwire/rokwire-building-blocks-api/compare/1.0.0...1.0.1
-[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
+[1.0.0]: https://github.com/rokwire/rokwire-building-blocks-api/releases/tag/1.0.0
```
Comment (sandeep-ps) [inline CHANGELOG.md:469]: I'm approving this PR. 
There is some unrelated change here. Please remove it if it can be fixed. Thanks.

**Review by sandeep-ps — APPROVED** [review]

**Comment by sandeep-ps** [comment]
Also, the branch is out-of-date with the base branch. Please fix it. Thanks.

**Comment by ywkim312** [comment]
Is this part of this #867 Why there are two PR for the same thing and both get merged to Develop?

**Review by ywkim312 — APPROVED** [review]
I am approving this and we can discuss more on the other PR

**Comment by sandeep-ps** [comment]
> Is this part of this #867? Why there are two PR for the same thing and both get merged to Develop?

The other PR was branched out of this. I will merge this and we can discuss it in the PR as you suggested. Thanks.

**Pull request merged**